valhalla/thor/costmatrix.h

```diff
@@ -302,11 +308,15 @@ class CostMatrix {
    * @param   time_infos   The time info objects for the sources
    * @param   invariant    Whether time is invariant
```

Comment:
forgot to update docstring to say that its returning the shape if it was requested